packages/Uploader/CHANGELOG.md

```diff
@@ -17,3 +17,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 
 - Updated: Created and placed all constants into type.js file. [@kaan.darcey](https://github.com/KDarcey).
+
+## [1.1.1-alpha.0] - 2020-12-16
```

Comment:
another small comment, you don't need to put the version and date here, you can just put everything under the unreleased section. The version actually should be  `1.1.1-alpha.1`. but even you put it here, the build won't fail, so it's ok

Reply:
@allison-c this was added automatically by running `yarn generateChangelog`... am i not supposed to run that?  I don't know about anyone else, but I'm getting confused by what is done manually, and what is done automatically (especially now that I'm working in sriracha, too, which has its own scripts) in the changelog and package.json version.

Package.json versions get updated automatically when deployment happens, right?  Maybe we can have the changelog get auto-generated at that point?

Reply:
Yes `yarn generateChangelog` will be run at that point. So we never need to do that manually

Reply:
ohhh not you shouldn't run it, changeLog runs automatically after the release :)